examples/all_in_one/run.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -euo pipefail
+
+DOCKER=${DOCKER:-docker}
+DOCKER_COMPOSE=${DOCKER_COMPOSE:-docker-compose}
+
+echo
+
+for req in DOCKER DOCKER_COMPOSE; do
+    if ! ${!req} -v >/dev/null 2>1 ; then
+        echo "Unable to run ${!req}, make sure that it's installed and working."
+        exit 1
+    fi
+done
+
+
+${DOCKER_COMPOSE} down -v
+${DOCKER_COMPOSE} up -d
+
+echo "---------------------------------------------------------------------------"
+echo "All services started, you can access them at the following addresses:"
+echo "Prometheus: http://localhost:9090/"
+echo "Grafana: http://localhost:3000"
+echo " User: admin, Password: admin"
+echo "---------------------------------------------------------------------------"
+echo "You can create SLO events by accessing:"
+echo "(Availability) SUCCESS events: http://localhost:8080/"
+echo "(Availability) FAIL events: http://localhost:8080/err"
+
+echo "For a start, this script will keep generating SLO events in randomized success/fail ratio. Check the Grafana dashboards by yourself to see how this impacts the defined SLO."
+while true; do
```

Comment:
this could be as an entrypoint of another docker-compose container with dependency on the already running nginx so nothing is really running on the host and all dependencies are ensured?

Reply:
Replied at https://github.com/seznam/slo-exporter/pull/4#discussion_r449196911